build.py

```diff
@@ -182,6 +182,8 @@ def core_cmake_args(components, backends, install_dir):
         cmake_enable('gcs' in FLAGS.filesystem)))
     cargs.append('-DTRITON_ENABLE_S3:BOOL={}'.format(
         cmake_enable('s3' in FLAGS.filesystem)))
+    cargs.append('-DTRITON_ENABLE_AZURE:BOOL={}'.format(
```

Comment:
AZURE is too generic, how about AZURE_STORAGE?

Reply:
changed to `TRITON_ENABLE_AZURE_STORAGE`